docs/faq/glossary.md

```diff
@@ -435,6 +441,10 @@ Online Travel Agency. OTAs are online platforms or websites that allow consumers
 research plan and book various travel-related services and products often offering a
 wide range of options and choices from different suppliers.
 
+## Partner Configuration
+
+The Partner Configuration allows Partners to make themselves known to the Camino Network as service providers, or as looking for services from other Partners in the Network. Upon matching between services wanted and offered by two parties, such services will be then traded on Camino Messenger.
+
 ## Partner plugin
 
 The Partner Plugin serves as a bridge enabling the Camino Messenger Bot to integrate
```

Comment:
```suggestion
The Partner Plugin serves as a bridge enabling the Camino Messenger Bot to integrate with the Partner distribution or inventory system.

More details can be found in our functional documentation: [Camino Messenger Bot Partner Plugin](https://docs.camino.network/partners/partner-plugin "Camino Messenger Bot Partner Plugin")
```

Reply:
@SamJaarsma this link is invalid:
https://docs.camino.network/partners/partner-plugin

Did you mean this one: https://docs.camino.network/camino-messenger/bot/partner-plugin/

Reply:
oops, correct!

Reply:
done